uaclient/system.py

```diff
@@ -178,6 +179,38 @@ def get_kernel_info() -> KernelInfo:
         )
 
 
+# This only works if we are root (because of permissions to 'file' the kernels
+# in /boot), but can't assert_root here due to circular imports.
```

Comment:
I understand we can't use the decorator, but we could do
```
if not util.we_are_currently_root():
    raise Exception()
```

Reply:
well well we don't reeeeally need the exception - when executed as non-root this returns an empty list. Do you think it is worth failing anyway?

Reply:
decided this should be a `RuntimeError`